apps/laboratory/src/components/Ethers/EthersGetCallsStatusTest.tsx

```diff
@@ -13,15 +13,20 @@ import { W3mFrameProvider } from '@reown/appkit-wallet'
 import { type GetCallsStatusParams } from '../../types/EIP5792'
 import { EIP_5792_RPC_METHODS } from '../../utils/EIP5792Utils'
 
-export function EthersGetCallsStatusTest() {
+export function EthersGetCallsStatusTest(params: { callsHash: string }) {
+  const callsHash = params.callsHash
   const [isLoading, setLoading] = useState(false)
-  const [batchCallId, setBatchCallId] = useState('')
+  const [batchCallId, setBatchCallId] = useState(callsHash)
```

Comment:
Why do we need another state variable.
Unless I'm missing something you can just use callsHash from the props ?

Reply:
because `batchCallId` can be set separately and is not always eql to `callsHash`